frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.tsx

```diff
@@ -215,6 +215,15 @@ export function getBucketListItem(
   };
 }
 
+const TriggerButton = forwardRef(function TriggerButton(
```

Comment:
This component is needed because we assigned it on line 220 in the previous version to `BaseBucketPickerPopover`, and to not keep adding to styled component (emotion).

Also the ref is needed because this component is used as a popover target.